symbols/ram.yml

```diff
@@ -428,9 +474,11 @@ ram:
       address:
         EU: 0x22AC258
         NA: 0x22AB918
+        JP: 0x22AD0D4
       length:
         EU: 0xC
         NA: 0xC
+        JP: 0x8
```

Comment:
Is this supposed to be shorter?

Also I just noticed that the struct declaration says this is 10 bytes long, which is somehow different than all the listed lengths? Can we fix this? We can use the `PMDSKY_VERSION` macro

Reply:
Yes, JP allocates less space for this symbol than the other versions.